src/test/java/fr/pilato/elasticsearch/tools/BeyonderRestIT.java

```diff
@@ -152,4 +156,29 @@ public void testAliases() throws Exception {
         Map<String, Object> response = asMap(client.performRequest("GET", "/_alias/foo"));
         assertThat(response, hasKey("test_aliases"));
     }
+
+    @Test
+    public void testMergeDisabled() throws Exception {
+        ElasticsearchBeyonder.start(client, "elasticsearch");
```

Comment:
According to the fact that the default path name is `elasticsearch`, I think we can call here instead:

```java
ElasticsearchBeyonder.start(client);
```

Not important but that will at least tell my IDE to stop proposing making `start(RestClient)` as a private method ;)

Reply:
Oh good point! I, too, get annoyed by that IDE hint, though I suppose it's just trying to nudge us into the TDD direction? :)